diff --git a/stm32mp157f-dk2.dts.orig b/stm32mp157f-dk2.dts
index c010473..7cbe626 100644
--- a/arch/arm/boot/dts/stm32mp157f-dk2.dts
+++ b/arch/arm/boot/dts/stm32mp157f-dk2.dts 

@@ -172,3 +172,18 @@
 	status = "okay";
 };
 
+&spi4 {
+	pinctrl-names = "default", "sleep";
+	pinctrl-0 = <&spi4_pins_b>;
+	pinctrl-1 = <&spi4_sleep_pins_b>;
+	status = "okay";
+	cs-gpios= <&gpioe 11 0>;
+		
+	spidev@0{
+	        compatible = "rohm,dh2228fv";	
+	        reg = <0>;  /* CS #0 */
+	        spi-max-frequency = <1000000>;
+	    };
+};
+
+
